blog/2024-11-25-identity-pool-with-aws-cognito.md

```
@@ -0,0 +1,154 @@
+## 소개
+
+Amazon Cognito는 웹 및 모바일 앱에 대한 사용자 인증 및 승인을 처리한다. 사용자 풀을 사용하면 앱에 가입 및 로그인 기능을 쉽고 안전하게 추가할 수 있다. Identity pools를 사용하면 앱은 사용자가 익명이든 인증되었든 특정 AWS 리소스에 대한 액세스 권한을 부여하는 임시 자격 증명을 얻을 수 있다.
+
+미국, 수술실이라는 특수한 환경에서 사용하는 애플리케이션을 개발하면서 사용자 요청을 최소화하기 위해 ML model 적용, 제스쳐 동작 등 다양한 시도를 하였다. 이번에는 사용자 인증과 관련하여 어떻게 하면 불필요한 과정을 줄이고 보다 안전하게 적용할 수 있을지 고민하며 Amazon Coginto를 적용한 내용이다.
+
+
+:::info 목차
+
+1. [User pool](#user-pool)
+2. [Identity pool](#identity-pool)
+3. [Cognito Sync](#github-action을-통한-vercel-수동-배포-완전-정복-with-모노레포)
```

Markdownlint (0.35.0)

Link fragments should be valid

(MD051, link-fragments)